docs/pages/access-controls/access-lists/guide.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Getting Started with Access Lists
+description: Learn how to use Access Lists to manage and audit long lived access to Teleport managed resources.
+---
+
+This guide will help you:
+- Create an access list
+- Assign a member to it
+- Verify permissions granted through the list membership
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- A running Teleport cluster with some built in resources. Following the
+  [Web Application Access Guide](../../application-access/guides/connecting-apps.mdx) will create a base with a
```

Comment:
The guide just tells you that you need some resources and kind of assumes that you don't have access to the resource before being added to the access list and that you do have access after being added. I wonder if there's a way to make this more concrete.

Reply:
I'm sure there is. What do you think about making the `Connecting Apps` guide a little more fully featured in this doc? We effectively need to instruct the user, "create a few protected resources" somehow.

Reply:
I've added in a step to add the dumper app to the cluster. Let me know what you think.